package.json

```diff
@@ -35,7 +35,7 @@
     "prettier": "^2.0.1",
     "replace-in-files-cli": "^1.0.0",
     "ts-jest": "^26.2.0",
-    "typescript": "^4.0.5"
+    "typescript": "4.0.5"
```

Comment:
Small wondering, not blocking or anything, but is the version lock needed?

Reply:
It seems like a problem with `ts-jest` not supporting TS 4.1. [I recorded a video to explain it](https://www.loom.com/share/e708cd1c072b4d9e8034a074e331aaf6) and I was planning to do a separate PR if you guys have seen the same in your branches.